modules/tinymce/src/models/dom/main/ts/table/api/TableResizeHandler.ts

```diff
@@ -138,32 +178,32 @@ export const TableResizeHandler = (editor: Editor): TableResizeHandler => {
         sz.on();
       }
 
-      sz.events.startDrag.bind((_event) => {
-        selectionRng.set(editor.selection.getRng());
-      });
+      bindEvents(sz);
+      tableResize.set(sz);
+    }
+  };
 
-      sz.events.beforeResize.bind((event) => {
-        const rawTable = event.table.dom;
-        Events.fireObjectResizeStart(editor, rawTable, Utils.getPixelWidth(rawTable), Utils.getPixelHeight(rawTable), barResizerPrefix + event.type);
-      });
+  editor.on('init', () => createResizeHandler());
 
-      sz.events.afterResize.bind((event) => {
-        const table = event.table;
-        const rawTable = table.dom;
-        Utils.removeDataStyle(table);
+  if (editor.inline) {
+    editor.on('NodeChange', () => {
```

Comment:
I'm not that confident that `NodeChange` will not trigger while you are dragging. Since that can be triggered by calling `editor.nodeChange()` and we some times call that our self to update UI state. So things like auto saving, async operations like spellchecking or link checking could potentially call that. Can't we use `mouseup` instead or similar since that will only happen when the user has finished the drag operation and I guess that mouse up would be on the blocker or bar not in the editor event like this? Doesn't dragster handle all that drag state, like startDrag, dragging and dragEnd?

Reply:
Thanks for highlighting. Now I wonder if this is the correct approach to tackle the bug. I'm working on another way of handling all the issues highlighted, which doesn't require changing the `z-index` or using `NodeChange` to remove the container and add that back.